openmsupply/application-manager-server
# #388 Laos Create Company template

First implementation here, still need to do:

- ~~pre-populate fields using defaults~~ (some done, others need further clarification across whole workflow)
- decide exactly which tables should be updated on Approval (currently ALL fields go to `organisation`, but some probably should go to `user`)
- permission_name "applyOrgLicence" doesn't actually exist yet, even though template grants it
- change `isLinear` to "true"
- have not created any org fields based on docs yet -- do we need to discuss how to handle?

Also done as part of this PR:

- add "01_users.js" to laos -- basically same as dev/users, but with generic applicant instead of our personal names
- updated the `/check-unique` endpoint so it can check any table/field combination (as well as existing "presets"). Also ensure it returns "true" if table or field doesn't exist yet
- bump evaluator package version to 1.8.0 in evaluator gui

### Testing:

- fresh install with `yarn database_init laos`
- log in as "applicant" (PW: 123456)
- Create company and submit
- Review as "userWithMultiplePermissions" -- Approve all and Submit
- Org should be created with all the new fields
